ui-admin/src/api/api.tsx

```diff
@@ -435,8 +452,31 @@ export default {
 
   async fetchEnrolleeChangeRecords(portalShortcode: string, studyShortcode: string, envName: string,
     enrolleeShortcode: string): Promise<DataChangeRecord[]> {
-    const url = `${baseStudyEnvUrl(portalShortcode, studyShortcode, envName)
-    }/enrollees/${enrolleeShortcode}/changeRecords`
+    const url =
+      `${baseStudyEnvUrl(portalShortcode, studyShortcode, envName)}/enrollees/${enrolleeShortcode}/changeRecords`
+    const response = await fetch(url, this.getGetInit())
+    return await this.processJsonResponse(response)
+  },
+
+  async createKitRequest(portalShortcode: string, studyShortcode: string, envName: string,
+                         enrolleeShortcode: string, kitType: string): Promise<string> {
+    const params = new URLSearchParams({ kitType })
+    const url =
+      `${baseStudyEnvUrl(portalShortcode, studyShortcode, envName)}/enrollees/${enrolleeShortcode}/requestKit?${params}`
+    const response = await fetch(url, { method: 'POST', headers: this.getInitHeaders() })
+    return await this.processJsonResponse(response)
+  },
+
+  async fetchKitRequests(portalShortcode: string, studyShortcode: string, envName: string,
```

Comment:
maybe rename to `fetchEnrolleeKitRequests` for clarity since we'll likely have a few of these (fetch all, fetch by study, etc..)?